RELEASE/scripts/autoscend/auto_post_adv.ash

```diff
@@ -169,13 +169,11 @@ void handlePostAdventure()
 		}
 	}
 
+
 	float regen = numeric_modifier("MP Regen Min").to_float() * 2.0;
 	regen += numeric_modifier("MP Regen Max").to_float();
 	regen = regen / 3.0;
 
-
```

Comment:
Try to avoid necessary whitespace when you see it, no need for a new PR/review though. Just one of those things I try to tidy up when i notice it.